ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualSignedTransaction.kt

```diff
@@ -44,41 +44,9 @@ interface ConsensualSignedTransaction {
     fun toLedgerTransaction(): ConsensualLedgerTransaction
 
     /**
-     * Sign the current [ConsensualSignedTransaction] with the specified key.
+     * Gets the missing signatories from the current [ConsensualSignedTransaction].
      *
-     * @param publicKey The private counterpart of the specified public key will be used for signing the
-     *      [ConsensualSignedTransaction].
-     * @return Returns the new [ConsensualSignedTransaction] containing the applied signature and the signature itself.
-     */
-    @Suspendable
-    fun addSignature(publicKey: PublicKey): Pair<ConsensualSignedTransaction, DigitalSignatureAndMetadata>
-
-    /**
-     * Adds a signature to the current [ConsensualSignedTransaction].
-     *
-     * @param signature The signature to be added to the [ConsensualSignedTransaction].
-     *
-     * @return Returns a new [ConsensualSignedTransaction] containing the new signature.
-     */
-    @Suspendable
-    fun addSignature(signature: DigitalSignatureAndMetadata): ConsensualSignedTransaction
-
-    /**
-     * Crosschecks the missing signatures with the available keys and signs the transaction with their intersection
-     * if there are any. (Disabled until crypto support becomes available.)
-     *
-     * @return Returns a new [ConsensualSignedTransaction] containing the new signature.
-     *
-    @Suspendable
-     * @return Returns the new [ConsensualSignedTransaction] containing the applied signature and a
-     *          list of added signatures.
-    fun addMissingSignatures(): Pair(ConsensualSignedTransaction, list<DigitalSignatureAndMetadata>)
-    */
-
-    /**
-     * Gets the signing keys for any missing transaction signatures.
-     *
-     * @return Returns a [Set] of [PublicKey] representing the signing keys for any missing transaction signatures.
+     * @return Returns a [Set] of [PublicKey] representing the missing signatories from the current [ConsensualSignedTransaction].
      */
     @Suspendable
     fun getMissingSignatories(): Set<PublicKey>
```

Comment:
I think we could move `getMissingSignatories` to an internal interface. If users cannot sign things themselves, then `getMissingSignatories` only benefits us.

Reply:
Can argue the same with `verifySignatures`.

Reply:
I agree with @lankydan. Anything that won't be called by a user can be internal/runtime-only.

Reply:
Do not we want to offer similar verification options for the users in ConsensualSignedTransactionVerifier?

Reply:
Anyway I have moved it to the internal interface for now.